tests/foreman/ui_airgun/test_provisioningtemplate.py

```diff
@@ -59,3 +76,78 @@ def test_positive_clone(session):
         template = session.provisioningtemplate.read(clone_name)
         assert set(os_list) == set(
             template['association']['applicable_os']['assigned'])
+
+
+@tier2
+@upgrade
+def test_positive_end_to_end(session, module_org, module_loc, template_data):
+    """Perform end to end testing for provisioning template component
+
+    :id: b44d4cc8-b78e-47cf-9993-0bb871ac2c96
+
+    :expectedresults: All expected CRUD actions finished successfully
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    name = gen_string('alpha')
+    new_name = gen_string('alpha')
+    os = entities.OperatingSystem().create()
+    host_group = entities.HostGroup(
+        organization=[module_org], location=[module_loc]).create()
+    environment = entities.Environment(
+        organization=[module_org], location=[module_loc]).create()
+    input_name = gen_string('alpha')
+    variable_name = gen_string('alpha')
+    template_input = [{
+        'name': input_name,
+        'required': True,
+        'input_type': 'Variable',
+        'input_content.variable_name': variable_name,
+        'input_content.description': gen_string('alpha')
+    }]
+    combination = [{
+        'host_group': host_group.name, 'environment': environment.name}]
+    with session:
+        session.provisioningtemplate.create({
+            'template.name': name,
+            'template.default': True,
+            'template.template_editor.editor': template_data,
+            'template.audit_comment': gen_string('alpha'),
+            'inputs': template_input,
+            'type.snippet': False,
+            'type.template_type': 'iPXE template',
+            'association.applicable_os.assigned': [os.title],
+            'association.hg_environment_combination': combination,
+            'organizations.resources.assigned': [module_org.name],
+            'locations.resources.assigned': [module_loc.name],
+        })
+        assert session.provisioningtemplate.search(name)[0]['Name'] == name
+        pt = session.provisioningtemplate.read(name)
+        assert pt['template']['name'] == name
+        assert pt['template']['default'] is True
```

Comment:
Is that intended? Why not just `assert smth/assert not smth` instead of `assert smth is True/assert smth is False`?

Reply:
yes, I prefer that style. In my opinion - it is more explicit and stable (remove any chance for `None` and `''`)

Reply:
If there is any serious point to do otherwise (except stylistic) - I am ready to do that from all my upcoming PRs